src/instructlab/model/download.py

```diff
@@ -220,10 +220,10 @@ def download(self):
         logger.debug(f"Running skopeo command: {command}")
 
         try:
-            subprocess.run(command, check=True, capture_output=True, text=True)
+            subprocess.run(command, check=True, text=True)
         except subprocess.CalledProcessError as e:
             click.secho(
-                f"Failed to run skopeo command: {e}.\nstdout: {e.stdout}.\nstderr: {e.stderr}",
+                f"Failed to run skopeo command: {e}. \nstderr: {e.stderr}",
```

Comment:
What was the reason for the space between the . and the \n ?

Reply:
@jaideepr97 was the original PR author

since that one was merged I'd like to keep the backport consistent

Reply:
no reason, probably just missed it